README.md

```diff
@@ -2,46 +2,61 @@
 
 # linkerd2-app chart
 
-This chart is based on the official linkerd2 helm chart. The main difference it offers is a
-direct integration with [cert-manager](https://cert-manager.io/) used to generate short-lived
-certificates for linkerd2 components. This results in an entirely hands-off install - no
-need to generate a CA certificate prior to installation.
+This chart is based on the official linkerd2 helm chart.
 
 ## Requirements
 
 - you can install only one release of this chart per kubernetes cluster.
-- `cert-manager` must already be deployed in the cluster.
 - it is strongly suggested to use the [`linkerd2-cni-app`](https://github.com/giantswarm/linkerd2-cni-app) as this results in a more secure setup.
   - the CNI must be installed before this chart, and `global.cniEnabled: true` must be set.
+- with linkerd CNI enabled, all pods meshed by linkerd require a `PodSecurityPolicy`, which allows use of `EmptyDir` volumes
+  - the injected linkerd-proxy container requires an `EmptyDir` volume
```

Comment:
sorry to be the party-pooper again, its two volumes.. one for the `proxy-init` initContainer and one for the `proxy` sidecar container

check https://github.com/giantswarm/linkerd2-app/blob/subtree-2.10/helm/partials/templates/_volumes.tpl

```suggestion
  - the injected linkerd-proxy container requires an `EmptyDir` volumes
```